### PR TITLE
[In-Product Analytics] update labels

### PR DESCRIPTION
**Pull request by thenamankumar**
Update label of frequency user char in user admin analytics. 

## Test plan

image

image

## App preview:

- [Web](https://sg-web-naman-update-labels-box.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/site-admin/analytics/AnalyticsUsersPage/index.tsx [hunk 1]
```diff
@@ -155,7 +155,7 @@ export const AnalyticsUsersPage: FC<RouteComponentProps> = () => {
                     {frequencies && (
                         <ChartContainer
                             title="Frequency of use"
-                            labelX="Days used"
+                            labelX="Minimum days used"
```
Comment (rrhyne) [inline client/web/src/site-admin/analytics/AnalyticsUsersPage/index.tsx:158]:
```suggestion
                            labelX="Minimum days by % of users"
```

Comment (thenamankumar) [reply client/web/src/site-admin/analytics/AnalyticsUsersPage/index.tsx:158]: the y axis says "Percentage of active users" for percentage and "Unique users" for totals as seen in the screenshot

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.60 kb) | 0.01% (+0.60 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/naman/update-labels-box/compare-report.html;tab=live_object) for a full comparison between the commits d04d076 and fb922dd or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by philipp-spiess — APPROVED** [review]
Code LGTM

File: client/web/src/site-admin/analytics/AnalyticsUsersPage/index.tsx (hunk 1 above)
Comment (rrhyne) [reply client/web/src/site-admin/analytics/AnalyticsUsersPage/index.tsx:158]: I see, please disregard my comment.

**Review by rrhyne — APPROVED** [review]
Thanks for making these changes so fast.

File: client/wildcard/src/components/Charts/components/bar-chart/BarChart.tsx [hunk 2]
```diff
@@ -133,6 +135,7 @@ export function BarChart<Datum>(props: BarChartProps<Datum>): ReactElement {
                         getDatumLink={getDatumLink}
                         onBarClick={handleBarClick}
                         onBarHover={onDatumHover}
+                        getDatumHoverValueLabel={getDatumHoverValueLabel}
```
Comment (vovakulikov) [inline client/wildcard/src/components/Charts/components/bar-chart/BarChart.tsx:138]: All this special props about labels and names makes me thing that we need to come up with more generic api for tooltip content like
```
tooltipContent={datum => <span>any tooltip content</span>}
```

I will try to look into this next week.

**Pull request merged**